mutation_query.cc

```diff
@@ -45,6 +45,15 @@ reconcilable_result::operator==(const reconcilable_result& other) const {
     return boost::equal(_partitions, other._partitions);
 }
 
+void
+reconcilable_result::merge(schema_ptr schema, const reconcilable_result& other) {
+    std::copy(other._partitions.begin(), other._partitions.end(), std::back_inserter(_partitions));
+    _short_read = _short_read || other._short_read;
```

Comment:
How can you merge something that comes after a short read? If you get a short read you have to stop.

Reply:
Yes, but I leave this to the caller.